web-share/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
```

Comment:
Oh OK it's more about the person reading the test, not the person manually executing it.

I'm not sure, in some sense this is just a language thing, you should generally be expected to know what the arguments are for a function and if you don't, you go look at the function (not all call sites need to document what the function args are). On the other hand, it would be easier to tell which ones matched which fields if they were dicts. What do you say about leaving the true/false cancel expectation as an argument, but moving all the ShareData expectations into a dictionary?